clang/test/Interpreter/crash.cpp

```diff
@@ -0,0 +1,20 @@
+// REQUIRES: host-supports-jit, x86_64-linux
+
+// RUN: rm -rf %t
+// RUN: mkdir -p %t
+//
+// RUN: split-file %s %t
+//
+// RUN: %clang++ -std=c++20 -fPIC -c %t/vec.cpp -o %t/vec.o
+// RUN: %clang++ -shared %t/vec.o -o %t/vec.so
+//
+// RUN: cat %t/Test.cpp | LD_LIBRARY_PATH=%t:$LD_LIBRARY_PATH clang-repl
```

Comment:
How does `clang-repl` respond to `#include <vector>` if the path to the library is not provided on the command line? IIUC, this will continue to search for a systemwide installation of the C++ standard library. If that's the case, this continues to be wrong.